sale_order_mass_action/wizards/sale_order_mass_action_wizard.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2015-Today GRAP (http://www.grap.coop)
+# @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+# Copyright 2022 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrderMassActionWizard(models.TransientModel):
+
+    _name = "sale.order.mass.action.wizard"
+
```

Comment:
a description is required, otherwise a warning is raised. Don't you think?

Reply:
Done. I had a warning on this